packages/playwright/src/tests/09-donate.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { test, expect } from '@playwright/test'
+
+import { getUrl } from '../utils'
+test.beforeEach(async ({ page }) => {
+  await page.goto(getUrl('kunsthalte'))
+  await page.waitForTimeout(1000)
+  await page.click('button:has-text("EN")')
+  await page.waitForTimeout(1000)
+})
+
+test.describe('01. Donate', () => {
+  test('TC-01: should Donate', async ({ page }) => {
+    await page.getByRole('link', { name: 'Donate' }).click()
+
+    await expect(page.getByRole('heading')).toContainText('Donate') // 01. Does the Donate page open?
+    const pageTitle = await page.title()
+    expect(pageTitle).toContain('Donate') // 02. Does the title match the page name?
+    await page.getByRole('button', { name: '€10', exact: true }).click()
+    const inputElement = page.locator('.chakra-numberinput input')
+    const value = await inputElement.inputValue()
+    expect(value).toContain('10')
+    await page.getByRole('button', { name: '€5', exact: true }).click()
+    await page.getByRole('button', { name: '€20' }).click()
+    await page.getByRole('button', { name: '€50' }).click()
+    await page.getByRole('button', { name: '€100' }).click()
+    await page.getByRole('button', { name: '€10', exact: true }).click()
+    await page.locator('.css-8pgw4r').first().click()
```

Comment:
Request testid or add TODO comment:

```suggestion
    // TODO: Add data-testid
    await page.locator('.css-8pgw4r').first().click()
```